apps/console/src/app/tenant-config.ts

```diff
@@ -5,7 +5,7 @@ export const consoleTenantConfig: TenantConfig[] = [
     host: 'localhost',
     application: 'console',
     tenant: 'me',
-    baseApiUrl: 'https://preprod-console-api.cme.openhbx.org',
+    baseApiUrl: 'http://localhost:3000',
```

Comment:
@TreyE  - is this change intentional ?

Reply:
Yes, this is for local host testing, which you should be doing with docker.